core/tauri-utils/src/config.rs

```diff
@@ -671,7 +671,7 @@ pub struct CliArg {
   pub required: bool,
   /// Sets an arg that override this arg's required setting
   /// i.e. this arg will be required unless this other argument is present.
-  #[serde(alias = "requred-unless-present")]
+  #[serde(alias = "required-unless-present")]
```

Comment:
This might be a breaking change. I don't know what it's doing.

There is support in `serde` for multiple things...

Reply:
should be fine